recipes/nsight-compute/meta.yaml

```diff
@@ -0,0 +1,64 @@
+{% set name = "nsight-compute" %}
+{% set version = "2022.4.0.15" %}
+{% set version_split = version.split(".")[0]+"."+version.split(".")[1]+"."+version.split(".")[2] %}
+{% set cuda_version = "12.0" %}
+{% set platform = "linux-x86_64" %}  # [linux64]
+{% set platform = "linux-ppc64le" %}  # [ppc64le]
+{% set platform = "linux-sbsa" %}  # [aarch64]
+{% set platform = "windows-x86_64" %}  # [win]
+{% set extension = "tar.xz" %}  # [not win]
+{% set extension = "zip" %}  # [win]
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://developer.download.nvidia.com/compute/cuda/redist/nsight_compute/{{ platform }}/nsight_compute-{{ platform }}-{{ version }}-archive.{{ extension }}
+  sha256: f084e05eb4d2ba32aceb25e1dcfe03f2a50127630973722b65219cf9e986a139  # [linux64]
+  sha256: 20e58ce79681bc8fd39394cfb8f8316c177fe4175af3ae95c025996f45904732  # [ppc64le]
+  sha256: 7d0b3d4d01ce36657fa739496c7b0a9c627f5fa42021c1696ddd15e119bb05a4  # [aarch64]
+  sha256: 958da9986841c49cb5a2885d1e14e4c673ba94e4b404ef9389d083b7a0095d84  # [win]
+
+build:
+  number: 0
+  skip: true  # [osx]
+  missing_dso_whitelist:
+    - "*"
```

Comment:
I assume conda-build fails without this? Just double-checking that we're not adding to the list unless really necessary.

Reply:
Yes, the libraries internal to `nsight-compute` linking each other are being flagged as missing, and the list is very long.

Reply:
Yeah let's try dropping this again. Think this may already be solved by dependencies added below. Even if that is not the case, would like to see if we can fix the dependencies up to make this unnecessary

Reply:
Looks like we need to keep this given the nature of the package. Marking resolved